### ISSUE TITLE
bug: ion-select multiple instances when selecting value on one it bleeds to another instance

### ISSUE THREAD

**Issue by fizdalf** [ionic-team/ionic-framework#22828]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**

**Current behavior:**

Having two ion-selects that pull from an Observable<any[]>, when I change the value of one, the value of the other changes, even though the assigned [value] property has not changed.
**Expected behavior:**

No other ion-select should change when I change the value of one.
**Steps to reproduce:**

I've created a stackblitz example depicting the issue https://stackblitz.com/edit/ionic-5-angular-10-start-template-zgakgg?file=src%2Fapp%2Ftab1%2Ftab1.page.html

Basically you create two ion-select assign a value from an unwrapped observable array, create a couple of options that will be shared by the instances, and create means of changing the array when an item is changed in any of the ion-selects...and you are good to go.
**Related code:**

```
insert short code snippets here
```

**Other information:**

**Ionic info:**

```
insert the output from ionic info here
```

**Comment by mhartington** [comment]
Hmm, so Im not 100% certain, but I think by having them subscribe to the same observable, it's binding the two together.

Quick test, can you remove Ionic from the equation? How does this behave  if you are just using a html select?

**Comment by fizdalf** [comment]
Hmmm, you are right, I did remove Ionic from the equation and I did get a similar behaviour. Not sure how to solve this then any word of advice?

**Comment by mhartington** [comment]
I'd check your observable flow. As it stands, it seems very complicated and could benefit from simplifying it a bit. Break down what you are trying to do and slowly add back some rx features if it's needed. Often you don't need much. 

Other than that, not much I could add. Im going to closing this since it's not an issue with Ionic specifically.
